R/fitNullModel.R

```diff
@@ -110,39 +125,6 @@ setMethod("fitNullModel",
           })
 
 
```

Comment:
moved this function into the nullModelInvNorm.R file with the other relevant functions